### PR TITLE
[liang] Fix compile warnings

### PR DESCRIPTION
**Pull request by Cphilo**

### CODE REVIEW

File: contracts/WooracleV2.sol [hunk 1]
```diff
@@ -185,7 +185,7 @@ contract WooracleV2 is InitializableOwnable, IWooracleV2 {
         timestamp = block.timestamp;
     }
 
-    function price(address base) external view override returns (uint256 price, uint256 priceTimestamp) {
+    function price(address base) external view override returns (uint256 _price, uint256 priceTimestamp) {
```
Comment (fb-alexcq) [inline contracts/WooracleV2.sol:188]: In general, '_price' is used for params or local variable. Better to rename it as 'priceOut'.

File: contracts/WooracleV2.sol [hunk 2]
```diff
@@ -223,8 +223,8 @@ contract WooracleV2 is InitializableOwnable, IWooracleV2 {
     }
 
     function isWoFeasible(address base) external view override returns (bool) {
-        uint256 price = uint256(infos[base].price);
-        return price != 0 && block.timestamp <= (timestamp + staleDuration);
+        uint256 _price = uint256(infos[base].price);
+        return _price != 0 && block.timestamp <= (timestamp + staleDuration);
```
Comment (fb-alexcq) [inline contracts/WooracleV2.sol:227]: How about:
```
  return infos[base].price != 0 && block.timestamp <= (timestamp + staleDuration);
```

File: contracts/WooracleV2.sol [hunk 3]
```diff
@@ -236,8 +236,8 @@ contract WooracleV2 is InitializableOwnable, IWooracleV2 {
     }
 
     // Wooracle price of the base token
-    function woPrice(address base) external view override returns (uint128 price, uint256 priceTimestamp) {
-        price = infos[base].price;
+    function woPrice(address base) external view override returns (uint128 _price, uint256 priceTimestamp) {
```
Comment (fb-alexcq) [inline contracts/WooracleV2.sol:239]: Same for the naming here.

**Review by fb-alexcq — CHANGES_REQUESTED** [review]
A few inline comments.

**Comment by Cphilo** [comment]
@fb-alexcq Resolved all comments.

**Review by fb-alexcq — APPROVED** [review]
Good!

**Pull request merged**